src/visualization/Turi Create Visualization/src/user_interface/src/elements/Explore/Evaluation/Components/TCEvaluationConfusionTable/index.js

```diff
@@ -0,0 +1,96 @@
+import React, { Component } from 'react';
+import './index.scss';
+
+import TCEvaluationConfusionRow from './TCEvaluationConfusionRow';
+import TCEvaluationConfusionHeader from './TCEvaluationConfusionHeader';
+
+class TCEvaluationConfusionTable extends Component {
+  
+  maxCount = () => {
+    return Math.max.apply(Math, this.props.considerations.map(function(o) { return o.count; }));
+  }
+
+  filterData = (element) => {
+    if(this.props.filter_confusion != null){
+      return element.actual == this.props.filter_confusion;
+    }else{
+      return true;
+    }
+  }
+
+  sortData = (a, b) => {
+    if(this.props.sort_by_confusions == "actual" && !this.props.sort_direction_confusions){
```

Comment:
It looks like this function can be simplified into:

```
const {sort_by_confusions, sort_direction_confusions} = this.props;
if (sort_by_confusions) {
  if (sort_direction_confusions) {
	return a[sort_by_confusions] < b[sort_by_confusions] ? -1 : 1;
  } else {
    return a[sort_by_confusions] > b[sort_by_confusions] ? -1 : 1;
  }
}
```

I also wonder if the currently code handles ties consistently.  Shouldn't equal return zero?